### ISSUE TITLE
Path for the config.gateway.json in unifi-docker (currently unsolved)

### ISSUE THREAD

**Issue by DirkWalter77** [jacobalberty/unifi-docker#308]
Synology DS216+
unifi-docker

I am looking for the right path to place the **config.gateway.json** within docker container (unifi-docker). I only have the following folder/files under /volume1/docker/Unifi/data

drwxrwxrwx+ 1   999   999    0 May  4 19:00 backup
-rwxrwxrwx+ 1   999   999 1322 May  4 19:00 system.properties
-rwxrwxrwx+ 1   999   999 1322 May  4 19:00 system.properties.bk

Uploading a map in Unifi-Controller won't create the folder sites/default.

Do anyone have a hint or a suggestion?

Best wishes!
Dirk

Synology 216+
Docker
Unifi

**Comment by buckaroogeek** [comment]
I use unifi-docker on a Synology DS218+. I use docker-compose to manage and
have the following volumes defined - these can be replicated via a docker
command too of course.

    volumes:
      - /volume1/docker/unifi/data:/unifi/data
      - /volume1/docker/unifi/log:/unifi/log
      - /volume1/docker/unifi/cert:/unifi/cert
      - /volume1/docker/unifi/init.d:/unifi/init.d

I have a config.gateway.json file in
/volume1/docker/unifi/data/sites/2u57o940 - the last segment of the path is
the only subdirectory under sites. I have not modified the json file in
months as I typically use it only for static hostname mapping for the local
home network and am shifting to using the pihole I also have running for
that purpose.  My docker-compose.yaml file can be explored and downloaded
from https://github.com/buckaroogeek/unifi-pihole-doh-docker

Brad

**Comment by DirkWalter77** [comment]
Hey Brad,

thanks for your answer.
Due your information I found my mistake. My problem was, that I did not had theses directories. My fault was, that I named the mounted folder "Unifi" with an "U" and not an "u" like "unifi". I deleted the container, installed it again with the mounted folder "unifi" and recovered the Unifi-Controller data from a backup. All the subdirectories were created then.

Thanks a lot!
Dirk
